package.json

```
@@ -54,6 +55,7 @@
     "cli-unit": "yarn unit-cli",
     "viewer-unit": "yarn unit-viewer",
     "watch": "yarn unit-core --watch",
+    "watch:report": "find report | entr yarn build-dist-reports",
```

```suggestion
    "watch:report": "find report | npx entr yarn build-dist-reports",
```

? (entr works for use b/c we have it globally installed)

(no strong feelings here but seems like this is simple and utility-enough to just keep in your own toolbox. like, we don't have a `"changed-files": "git diff"` ya know?)

---

sure lets just leave the entr stuff out of our scripts for now. i'll keep it to myself.

re the thread a bit higher up... i still think `build-dist-reports` is a reasonable script. it helps me hugely and compared to the nesting-ness of our npm scripts, it fits right in. so lets keep that, ok?

---

sgtm

---

i meant to remove watch:report but never did. doing now.